ui-tests/cypress/integration/administration/labels.spec.js

```diff
@@ -46,6 +53,18 @@ describe('Labels', () => {
         currentPage.getContent().getLanguageTextField('it').should('exist').and('be.visible');
     });
 
+    //SMOKE TEST
+    //ENG-3238
+    it('Action context menu', () => {
+        cy.labelsController().then(controller => controller.addLabel(testLabel.code, testLabel.name));
+        cy.wrap(testLabel).as('labelToDelete');
```

Comment:
you should concatenate this to the API call, as if it fails you'll have no label to be deleted

Reply:
Would it be okay like this?
`cy.labelsController().then(controller => {
            controller.addLabel(testLabel.code, testLabel.name);
            cy.wrap(testLabel).as('labelToDelete');
        })`

Reply:
you need to concatenate an additional then on top of the API on call

Reply:
Done